python/src/ecole/core/reward.cpp

```diff
@@ -167,6 +168,19 @@ void bind_submodule(py::module_ const& m) {
 
 		The difference in solving time is computed in between calls.
 		)");
+	
+	auto dualbound = py::class_<DualBound>(m, "DualBound", R"(
+		dual bound.
```

Comment:
This is not useful in the documentation. Perhaps "The problem global dual bound."
We could also add whether or not the is differential or a metric (see main comment in the PR discussion)